src/EnduranceTests/Targets/AzureBlobErrorOnlyTarget.cs

```diff
@@ -0,0 +1,54 @@
+﻿
+namespace NServiceBus.AzureServiceBus.EnduranceTests.Targets
+{
+    using System.Diagnostics;
+    using Microsoft.WindowsAzure.Storage;
+    using Microsoft.WindowsAzure.Storage.Blob;
+    using NLog;
+    using NLog.Targets;
+    using NServiceBus.AzureServiceBus.EnduranceTests.TestUtils;
+    using LogLevel = NLog.LogLevel;
+
+    public class AzureBlobErrorOnlyTarget : TargetWithLayout
+    {
+        private CloudBlobClient _client;
+
+        public CloudStorageAccount StorageAccount { get; set; }
+
+        protected override void InitializeTarget()
+        {
+            base.InitializeTarget();
+
+            _client = StorageAccount.CreateCloudBlobClient();
+        }
+
+        protected override void Write(LogEventInfo logEvent)
+        {
+            if (logEvent.Level < LogLevel.Error || logEvent.Exception == null) return;
+
+            if (_client == null) return;
```

Comment:
I just removed it and refactored some of the other initialization. It will fail much earlier.